hw_6n.py

```diff
@@ -0,0 +1,32 @@
+# 1. Створити телефонну книгу, яка матиме наступні команди:
+# stats: кількість записів
+# add: додати запис
+# delete <name>: видалити запис за іменем (ключем)
+# list: список всіх імен в книзі
+# show <name>: детальна інформація по імені
+# Записи не мають повторюватися, заборонено перезаписувати записи, тільки видаляти і додавати заново.
+
+phone_book = {'Vadym':'0665699589'}
+enter = input('''You can see all contacts(stats), all names(list), someone\'s number(show), (add) or (delete) contacts. 
```

Comment:
Whap the code starting from this line to `while True:` to be able to input more than one command while your program is running